.github/workflows/ci.yml

```
@@ -260,3 +260,106 @@ jobs:
       # when imported in an NPM node_modules tree.
       - name: Prepack packages
         run: yarn lerna run --reject-cycles --concurrency 1 prepack
+
+  test-xs:
+    name: test-xs
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [20.x]
+        moddable-version: [5.0.0]
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # without this, setup-node errors on mismatched yarn versions
+      - run: corepack enable
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      # end macro
+
+      - name: Run yarn build
+        run: yarn build
+
+      - name: Get XS
```

Here's an idea:

```suggestion
      - name: Check release
        id: check-release
        run: |
          if curl -f -s -o /dev/null -I -L https://api.github.com/repos/Moddable-OpenSource/moddable/releases/tags/${{ matrix.moddable-version }} then
              echo "release=true"
          else
              echo "release=false"
          fi >> $GITHUB_OUTPUT
      
      - name: Get XS
        if: steps.check-release.outputs.release == 'true'
```

With the build step if it's not.